modules/cnv-removing-virtio-disk-from-vm.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-existing-windows-vm.adoc
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-new-windows-vm.adoc
+
+[id="cnv-removing-virtio-disk-from-vm_{context}"]
+= Removing the VirtIO container disk from a virtual machine
+
+After you have installed all required VirtIO drivers to the virtual machine, the
+ `virtio-win` container disk no longer needs to be attached to the virtual machine. 
+Remove the `virtio-win` container disk from the virtual machine configuration file. 
+
+.Procedure
+. Edit the virtual machine configuration in the web console, or edit the 
```

Comment:
I've changed it so that is just 'edit the config', and included the $ oc edit vmi <vmi> command before the config snippet.
When the Web Console content for 1.4 (covers start, stop, restart, edit, etc) is available, I can xref to them as 'See Also' in the assemblies